test/extended/openstack/cpms.go

```diff
@@ -43,13 +43,9 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] ControlPlane Mach
 		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
 		controlPlaneMachineSet, numCpms, err = getControlPlaneMachineSet(ctx, dc)
 		if err != nil {
-			if numCpms == 0 {
```

Comment:
this was not useful. in case of an error, `numCpms` will always be `0`. Let's just skip the test if CPMS can't be found (in case of UPI for example).

Reply:
Is it possible that due to a bug there will be more than one CPMS?